packages/mrm-task-eslint/index.js

```diff
@@ -8,6 +8,29 @@ const {
 	getExtsFromCommand,
 } = require('mrm-core');
 
+const normalizePresetPackageName = presetName => {
+	const prefix = 'eslint-config';
+	const presetNameRegex = /^(?:(@[^/]+)\/?)?((?:eslint-config-)?[^/]*)(?:\/[^/]+)?$/;
+	const match = presetName.match(presetNameRegex);
+
+	if (!match) {
+		throw new Error('Invalid preset name');
+	}
+
+	const scope = match[1] || '';
+	let configName = match[2];
+
+	if (!scope && !configName.startsWith(prefix)) {
+		configName = `${prefix}-${configName}`;
+	} else if (scope && !configName) {
+		configName = prefix;
+	}
+
+	const packageName = `${scope ? `${scope}/` : ''}${configName}`;
+
+	return packageName;
+};
```

Comment:
Let's extract this piece to a function to avoid reassignment:

```js
const [, scope = '', configNameRaw] = match;

// ...

const getConfigName = (configName, scope, prefix) => {
if (!scope && !configName.startsWith(prefix)) {
		return `${prefix}-${configName}`;
	} else if (scope && !configName) {
		return prefix;
	} else {
return configName;
}
}
```